lib/webhook.js

```diff
@@ -21,9 +21,7 @@ module.exports = (bot, opt) => {
 
   // Check port
   if (!PORTS.includes(port)) {
-    let error = `[bot.error.webhook] allowed ports: ${ PORTS.join(', ') }`;
-    this.event('error', { error });
-    console.error(error);
+    console.error(`[bot.warning.webhook] allowed ports: ${ PORTS.join(', ') }`);
     return;
```

Comment:
If we remove error event from this block, then we will not be able to do any actions (this will not trigger main *telebot.js* error handler).